hazelcast/src/test/java/com/hazelcast/internal/json/JsonValue_Test.java

```diff
@@ -84,7 +86,7 @@ public void run() throws IOException {
   @Test
   public void writeTo_doesNotCloseWriter() throws IOException {
     JsonValue value = new JsonObject();
-    Writer writer = spy(new StringWriter());
+    Writer writer = mock(StringWriter.class);
```

Comment:
What would be the alternative? Add opens for the `java.io` package?

Reply:
Yes, although since we're testing only interaction here (writer not being closed) I think mock is acceptable